server/router/routes/index.js

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2021 Uber Technologies Inc.
```

Comment:
moved `server/router.js` route handlers into separate files in `server/router/routes`